src/config/types.ts

```diff
@@ -18,12 +19,33 @@ export type LayerType =
   | PointDataLayerProps;
 
 export type LayerKey = keyof typeof rawLayers;
+
+type MenuGroupItem = {
+  id: string;
+  label: string;
+  main: boolean;
+};
+
+export type MenuGroup = {
+  groupTitle: string;
+  activateAll: boolean;
+  layers: MenuGroupItem[];
+};
+
 /**
- * Check if a string is an explicitly defined layer in layers.json
- * @param layerKey the string to check
+ * Check if a string/object is an explicitly defined layer in layers.json
+ * @param layerKey the string/object to check
  */
-export const isLayerKey = (layerKey: string): layerKey is LayerKey =>
-  layerKey in rawLayers;
+export const isLayerKey = (layerKey: string | MenuGroup) => {
+  if (typeof layerKey === 'string') {
+    return layerKey in rawLayers;
+  }
+  if (typeof layerKey === 'object') {
```

Comment:
you probably don't need to check the type here since it will be a MenuGroup if not a string

Reply:
so, remove `if` and `return false`, make it like this?

```
if (typeof layerKey === 'string') {
   return layerKey in rawLayers;
}
-  if (typeof layerKey === 'object') {
-    const layers = map(layerKey.layers, 'id');
-    return every(layers, layer => layer in rawLayers);
-  }
-  return false;

+  const layers = map(layerKey.layers, 'id');
+  return every(layers, layer => layer in rawLayers);
 };
```

Reply:
yes exactly, and maybe just add a quick comment :)

Reply:
I am tempted to simplify this logic by creating the groups in `layers.json` instead of `prism.json`. Maybe something to consider for the future (not in this PR), cc @wadhwamatic

Reply:
from a configuration perspective, I think I prefer to keep it in prism.json. It fits logically there IMO

Reply:
dunno, I feel like calling `sdi_group` instead in prism.json would be quite convenient too. But we can revisit this later